kube_log_watcher/main.py

```diff
@@ -281,7 +285,12 @@ def watch(containers_path, agents_list, cluster_id, interval=60, kube_url=None,
     watcher_config = load_watcher_config(watcher_config_file)
 
     configuration = dict(watcher_config, cluster_id=cluster_id)
-    agents = load_agents(agents_list, configuration)
+
+    try:
+        agents = load_agents(agents_list, configuration)
+    except Exception:
+        logger.exception('Failed to load agents')
```

Comment:
Unhandled exception will be captured by sentry -> you maybe don't need this try-except at all